exchange_server/assets/dashboards/overview.json

```
@@ -32,22 +32,32 @@
                 "viz": "timeseries",
                 "requests": [
                     {
-                        "q": "avg:exchange.database.io_writes_avg_latency{*}",
+                        "q": "avg:exchange.database.io_writes_avg_latency{*} by {instance,host}",
```

Sorry, what I meant by template variables here is:
```suggestion
                        "q": "avg:exchange.database.io_writes_avg_latency{$instance,$host} by {*}",
```

https://docs.datadoghq.com/dashboards/template_variables/